sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -118,7 +119,9 @@ extern "C" {
  *          </a>.
  */
 typedef struct {
+#ifdef MODULE_GNRC_NDP_NODE
```

Comment:
It is already reflected (in #3748, that introduces 6LoWPAN-ND): https://github.com/RIOT-OS/RIOT/pull/3748/files#diff-cc423206e4b071a735d298217a657675R92